String/LengthofLongestSubstringWithoutRepetition.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @function lengthOfLongestSubstring
+ * @description Finds the length of the longest substring in a given string without repeating characters
+ * @param {String} str - The input string
+ * @returns {Number} The Length of the longest substring in a given string without repeating characters
+ * @example lower("abcabcbb") => 3
+ * @example lower("bbbbb") => 1
+ */
+
+const lengthOfLongestSubstring = (s) => {
```

Comment:
This could use some comments.

Reply:
Added

Reply:
I meant in the code, not a leetcode link (the leetcode link should also use the `@see` annotation).